code/game/machinery/telecomms/presets.dm

```diff
@@ -313,6 +313,9 @@ GLOBAL_LIST_EMPTY(all_static_telecomms_towers)
 	if(corrupted)
 		return
 
+	if(!inoperable() || on)
```

Comment:
```suggestion
	if(operable())
```
Not sure I see the point of testing `on` we only care about whether its operable.